Patch Initiative rolls out of Lucky Feat

Turtlelord26 opened the pull request:
Now enumerated use cases for Lucky, not including Initiative, per Issue #167 . Wording touched up to match enumeration.

I specifically ask for the sake of Luck >= 8 whether we can introduce the phrase Roll with Advantage to CX.

Turtlelord26 committed:
Patch Initiative rolls out of Lucky Feat

Now enumerated use cases for Lucky, not including Initiative, per Issue #167 . Wording touched up to match enumeration.

I specifically ask for the sake Luck >= 8 whether we can introduce the phrase Roll with Advantage to CX.

1sourcecontrol commented:
>I specifically ask for the sake Luck >= 8 whether we can introduce the phrase Roll with Advantage to CX.
@Turtlelord26 

Can you rephrase your sentence? I think you are asking if we can do, ">=" on stuff, but I want to be sure

Turtlelord26 commented:
I specifically ask, for the sake of Luck >= 8 ^cases^, **whether we can introduce the phrase Roll with Advantage** to CX. @1sourcecontrol

bleehu commented:
@1sourcecontrol I like the idea of adding "roll with advantage" and "roll with disadvantage" to CX. While I'm quite proud of how different CX is from DnD, I'm not so proud that I won't steal what's good from them. And I'm pretty sure there's no legal reason not to.

Turtlelord26 committed:
Roll with Advantage Terminology in Lucky Feat

It only really works with the 8 and 9 cases. Is that enough to justify it here?

Turtlelord26 committed:
Advantage definition

Explicit rules support for Advantage, including edge case handling.

Turtlelord26 committed:
Merge pull request #180 from bleehu/Turtlelord26-Issue_167-Patch_Feat_Lucky-Advantage_Wording

Roll with Advantage Terminology in Lucky Feat

bleehu reviewed (approved):
shipit!

Turtlelord26 commented:
@Wildlovelies As the originator of the Issue I'll be waiting on word from you on these changes.

1sourcecontrol commented on Basic Rules.txt:
```diff
@@ -43,6 +43,14 @@ which dice represents the tens place before you roll. i.e. if one die rolls a
 die which represents your ten's place and a purple die which represents your one's
 place, and you roll an orange 3 and a purple 9 then you rolled 39%.
 
+Sometimes circumstances or the occasional feat may prompt you to make a roll with
```
Can you add a header to this so we can easily find the rule? Such as:

>==== Rolling with Advantage/Disadvtange====

or

>Rolling with Advantage/Disadvantage:

bleehu replied:
Bleehu Likes this post!

1sourcecontrol commented on CharacterCreation/03_Feats.txt:
```diff
@@ -545,29 +545,28 @@ Lucky:
-		You can choose to reroll a die, four times per long rest. You can reroll
-		as many times as you want (which uses up your rerolls-per-rest) and if you 
-		reroll, choose which of the rolls to keep.
+		You may choose to reroll a check, save, or attack four times per long rest. 
```
Isn't it supposed to be "role with advantage", to keep the trend in terminology?

bleehu replied:
Reroll means you must take the second roll regardless of whether the second roll is superior or not. I don't see the need to formally define reroll, since it's a word in the English language.

Turtlelord26 replied:
The problem with using Advantage specifically in this spot is that in that, with advantage terminology, this subfeat is allowing you to choose whether to make a roll with no advantage, advantage, double advantage, or triple advantage, and your choice makes a difference to how many times you can do it again. The original phrasing is more clear.

1sourcecontrol replied:
@bleehu the wording here is supposed to be saying a fancy kind of "rolling with advantage"... unless you meant to clarify something else?

@Turtlelord26 , thanks, that makes sense.

bleehu replied:
@1sourcecontrol 
Reroll:
take the 2nd roll
Advantage:
take the higher roll

1sourcecontrol reviewed (changes requested):
oops, I approved without checking to see if you added the header

Turtlelord26 committed:
Advantage Subheader

Also removed unnecessary language as we don't have any rolls where lower is better.
